<?xml version="1.0"?>
<launch>
    <!-- param -->
    <arg name="robot_param" default="$(find point_follow_planner)/config/robot_param.yaml"/>
    <arg name="hz" default="5"/>
    <!-- topic name -->
    <arg name="robot_outline" default="/robot_outline"/>

    <node pkg="point_follow_planner" type="robot_outline_creator" name="robot_outline_creator" ns="local_planner" output="screen">
        <rosparam command="load" file="$(arg robot_param)"/>
        <param name="hz" value="$(arg hz)"/>
        <remap from="/robot_outline" to="$(arg robot_outline)"/>
    </node>
</launch>
